### ISSUE TITLE
error: unrecognized arguments: --dataset amazon-book

### ISSUE THREAD

**Issue by Sylvia-Huang1997** [LunaBlack/KGAT-pytorch#13]
I follow the steps, why is this displayed?

![image](https://user-images.githubusercontent.com/59968322/127771724-02041b41-c2f6-457b-8dac-6b5c81a4872a.png)

And this is my conda list:
numpy                     1.13.1
dgl-cu101                 0.6.1 
pandas                    0.23.4
scikit-learn              0.20.4
scipy                     1.5.4 
torch                     1.6.0+cu101

**Comment by San-ctuary** [comment]
you should replace the --dataset with --data_name

**Comment by ELFAHIM96** [comment]
usage: DETR training and evaluation script [-h] [--lr LR] [--lr_backbone LR_BACKBONE] [--batch_size BATCH_SIZE] [--weight_decay WEIGHT_DECAY] [--epochs EPOCHS] [--lr_drop LR_DROP]
                                           [--clip_max_norm CLIP_MAX_NORM] [--frozen_weights FROZEN_WEIGHTS] [--backbone BACKBONE] [--dilation]
                                           [--position_embedding {sine,learned}] [--enc_layers ENC_LAYERS] [--dec_layers DEC_LAYERS] [--dim_feedforward DIM_FEEDFORWARD]
                                           [--hidden_dim HIDDEN_DIM] [--dropout DROPOUT] [--nheads NHEADS] [--num_queries NUM_QUERIES] [--pre_norm] [--masks] [--no_aux_loss]      
                                           [--set_cost_class SET_COST_CLASS] [--set_cost_bbox SET_COST_BBOX] [--set_cost_giou SET_COST_GIOU] [--mask_loss_coef MASK_LOSS_COEF]     
                                           [--dice_loss_coef DICE_LOSS_COEF] [--bbox_loss_coef BBOX_LOSS_COEF] [--giou_loss_coef GIOU_LOSS_COEF] [--eos_coef EOS_COEF]
                                           [--dataset_file DATASET_FILE] [--data_path DATA_PATH] [--coco_panoptic_path COCO_PANOPTIC_PATH] [--remove_difficult]
                                           [--output_dir OUTPUT_DIR] [--device DEVICE] [--seed SEED] [--resume RESUME] [--start_epoch N] [--eval] [--num_workers NUM_WORKERS]      
                                           [--world_size WORLD_SIZE] [--dist_url DIST_URL]
DETR training and evaluation script: error: unrecognized arguments: data_path ../datasets/

**Comment by ELFAHIM96** [comment]
**$ $ python main.py --dataset_file face data_path ../datasets/ --output_dir output**
I get  This Error 
I have I problem in ### data_path 
![image](https://user-images.githubusercontent.com/65721811/160190206-e0133a9e-2a48-41aa-972e-860bac9bf351.png)
